src/CalendarStrip.js

```diff
@@ -409,6 +412,8 @@ class CalendarStrip extends Component {
       customDatesStyles: this.props.customDatesStyles,
       markedDates: this.props.markedDates,
       size: this.state.dayComponentWidth,
+      dayComponentHeight: this.state.dayComponentHeight,
+      height: this.state.height,
```

Comment:
@peacechen you mean this here? I think it can be removed (sorry its been a while since I made this PR)